index.html

```diff
@@ -5,222 +5,219 @@
-                                <li class="stack3">JavaScript</li>
-                                <li class="stack3">Ruby</li>
-                                <li class="stack3">HTML</li>
-                                <li class="stack3">CSS</li>
-                            </ul>
                             <span class="stack-title">
-                                <img src="assets/about/Rectangle_56.png" alt="shape">
-                                <h3>Frameworks</h3>
+                                <h3><img class="bullet" src="assets/about/Rectangle_56.png" alt="shape"> Frameworks</h3>
+                                <ul class="lang-frameworks2">
+                                    <li class="stack3">React.js</li>
+                                    <li class="stack3">Ruby on Rails</li>
+                                    <li class="stack3">RSpec</li>
+                                    <li class="stack3">Capybara</li>
+                                </ul>
                             </span>
-                            <ul class="lang-frameworks">
-                                <li class="stack3">React.js</li>
-                                <li class="stack3">Ruby on Rails</li>
-                                <li class="stack3">RSpec</li>
-                                <li class="stack3">Capybara</li>
-                            </ul>
                             <span class="stack-title">
-                                <img src="assets/about/Rectangle_57.png" alt="shape">
-                                <h3>Skills</h3>
-                            </span>
-                            <ul class="lang-frameworks">
-                                <li class="stack3">Database Management</li>
-                                <li class="stack3">Version Control</li>
-                                <li class="stack3">CLI</li>
-                                <li class="stack3">API Design</li>
-                                <li class="stack3">Web Development</li>
-                            </ul>
+                                <h3><img class="bullet" src="assets/about/Rectangle_57.png" alt="shape"> Skills</h3>
+                                <ul class="lang-frameworks3">
+                                    <li class="stack3">Database Management</li>
+                                    <li class="stack3">Version Control</li>
+                                    <li class="stack3">CLI</li>
+                                    <li class="stack3">API Design</li>
+                                    <li class="stack3">Web Development</li>
+                                </ul>
+                            </span>                           
                         </ul>
-                    </article>
+                    </div>>
                 </article>
+            </section>
+            <section class="feeback">
                 <article class="feedback">
                     <p class="feedback-text">
```

Comment:
👍 Great job on adding the heading! However, there are a few adjustments that still need to be made:

- [ ] Firstly, since this is a title, it would be better to use a proper title tag, such as h2 to h6, based on the structure of your project title. This will improve the overall project structure. Please consider updating your code accordingly. 🏗️📝

- [ ] While you have chosen the appropriate colors, font family, font-weight, the font size and line height do not match the design. Please take the design specifications into account and update your code accordingly. 🆎📐

Please take these recommendations into consideration to improve the alignment between your code and the design. Thank you for your attention to detail! 🙏😊

Reply:
Please clarify on the title you've mentioned here @shahadat3669